atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnSelection.java

```diff
@@ -30,7 +29,7 @@
 import com.google.common.primitives.UnsignedBytes;
 import com.palantir.atlasdb.encoding.PtBytes;
 
-public class ColumnSelection implements Serializable {
+public final class ColumnSelection implements Serializable {
```

Comment:
http://checkstyle.sourceforge.net/config_design.html#FinalClass

When constructors are private, it's impossible to extend the class as you can't call the constructor. Therefore, it makes sure that we mark them as final. It's a rule in baseline enabled by default